### PR TITLE
Feature/search sorting directory

### PR DESCRIPTION
**Pull request by miguel-merlin**
#128

### CODE REVIEW

File: @types/nextjs-routes.d.ts [hunk 1]
```diff
@@ -11,12 +11,12 @@ declare module "nextjs-routes" {
   } from "next";
 
   export type Route =
+    | StaticRoute<"/Directory">
```
Comment (ulrokx) [inline @types/nextjs-routes.d.ts:14]: Run `npx nextjs-routes` to regenerate this with lowercase

**Comment by ulrokx** [comment]
Not sure if this is just me 
image

**Review by ulrokx — CHANGES_REQUESTED** [review]
Looks good, just the types file

**Comment by miguel-merlin** [comment]
Ready to merge

**Review by ulrokx — APPROVED** [review]

**Pull request merged**